### PR TITLE
Reverses Announcment sound pref to hopefully fix it starting off.

### LINKED ISSUE #15443: announcements don't make sound anymore

### PR DESCRIPTION
**Pull request by Shadowlight213**
fixes #15443

### CODE REVIEW

File: code/modules/mob/living/silicon/ai/say.dm [hunk 1]
```diff
@@ -157,7 +157,9 @@ var/const/VOX_DELAY = 600
 		if(!only_listener)
 			// Play voice for all mobs in the z level
 			for(var/mob/M in player_list)
-				if(M.client && !M.ear_deaf && (M.client.prefs.toggles & SOUND_ANNOUNCMENTS))
+				if(M.client && !M.ear_deaf)
```
Comment (neersighted) [inline code/modules/mob/living/silicon/ai/say.dm]: Please keep these ifs nested. Just negate the conditional

File: code/modules/client/preferences_toggles.dm [hunk 2]
```diff
@@ -285,6 +285,6 @@ var/global/list/ghost_orbits = list(GHOST_ORBIT_CIRCLE,GHOST_ORBIT_TRIANGLE,GHOS
 	set category = "Preferences"
 	set desc = ".Toggles hearing Central Command, Captain, VOX, and other announcment sounds"
 	prefs.toggles ^= SOUND_ANNOUNCMENTS
-	src << "You will now [(prefs.toggles & SOUND_ANNOUNCMENTS) ? "hear announcment sounds" : "no longer hear announcments"]."
+	src << "You will now [(prefs.toggles & SOUND_ANNOUNCMENTS) ? "no longer hear announcments" : "hear announcment sounds"]."
```
Comment (tkdrg) [inline code/modules/client/preferences_toggles.dm]: it's `announcement` not announcment

File: code/modules/mob/living/silicon/ai/say.dm (hunk 1 above)
Comment (tkdrg) [inline code/modules/mob/living/silicon/ai/say.dm]: makes sense i guess but doesnt really matter

File: code/modules/mob/living/silicon/ai/say.dm [hunk 3]
```diff
@@ -157,7 +157,9 @@ var/const/VOX_DELAY = 600
 		if(!only_listener)
 			// Play voice for all mobs in the z level
 			for(var/mob/M in player_list)
-				if(M.client && !M.ear_deaf && (M.client.prefs.toggles & SOUND_ANNOUNCMENTS))
+				if(M.client && !M.ear_deaf)
+					if(M.client.prefs.toggles & SOUND_ANNOUNCMENTS)
+						return
```
Comment (YotaXP) [inline code/modules/mob/living/silicon/ai/say.dm]: Shouldn't this be `continue`?

**Pull request closed without merging**